lua/mapvote/rtv.lua

```diff
@@ -1,13 +1,15 @@
 RTV = RTV or {}
```

Comment:
This alone would fix it - I'll let @plally decide
```suggestion
RTV = RTV or {
    HandleRTVCommand = function() end,
    HandleUnRTVCommand = function() end
}
```